source/user-manual/capabilities/vulnerability-detection/how_it_works.rst

```diff
@@ -26,16 +26,17 @@ Scan types
 ^^^^^^^^^^
 
 The Vulnerability Detector module can run a scan on startup (:ref:`run_on_start <vuln_det_run_on_start>`) and every certain period of time (:ref:`interval <vuln_det_interval>`).
-In any of these cases, the packages that have already been scanned will wait until the :ref:`ignore_time <vuln_det_ignore_time>` expires to be re-scanned.
-This improves the performance and avoids repeated alerts during a configurable time. We have then two different types of scan:
+In any of these cases, the packages and operating system that have already been scanned will be re-scanned if the vulnerability database receives new CVEs information and :ref:`min_full_scan_interval <vuln_det_min_full_scan_interval>` expires.
+This improves the performance and avoids repeated alerts during a configurable time. We have then three different types of scan:
 
-- Full scan: The first time, Vulnerability Detector scans every single package installed. After this, all the available packages are scanned again only when the configured :ref:`ignore_time <vuln_det_ignore_time>` expires.
-- Partial scans: Only new packages are scanned while :ref:`ignore_time <vuln_det_ignore_time>` is still valid.
+- Baseline: This scan type will be triggered the first time after Vulnerability Detector is enabled, it performs a full scan considering every single package installed as well as the operating system without alerting any of those events.
+- Full scan: In this scan type, all the available packages are scanned and alerted only when the configured :ref:`min_full_scan_interval <vuln_det_min_full_scan_interval>` expires and the CVEs database is updated with new information.
+- Partial scans: Only new packages are scanned and alerted.
 
 There are few considerations that arise from this behavior:
 
-- Every full scan generates alerts for all the packages, so the alerts are repeated until they get fixed.
-- The user ca not trigger a full scan manually, the only option is to decrease the :ref:`ignore_time <vuln_det_ignore_time>` setting.
+- Full and partial scans will generate alerts for all the new vulnerabilities found after the CVEs database is updated.
```

Comment:
I suggest removing this line. Now, this is explained in the result of each scan type.
```suggestion
```